deeds-dapp-webapp/src/main/webapp/skin/less/deeds.less

```diff
@@ -306,6 +306,23 @@ h4 {
   left: -50px;
 }
 
+.my-menu {
+  margin-top: 10px;
+  contain: initial;
+  overflow: visible;
+}
+.my-menu::before {
+  position: absolute;
+  content: "";
+  right: 50%;
+  transform: translateY(-100%);
+  width: 10px; 
+  height: 13px; 
+  border-left: 10px solid transparent;
+  border-right: 10px solid transparent;
+  border-bottom: 13px solid #fff;
+}
+
```

Comment:
`my-menu` to replace with better name and we should IMO do it without a specific class